### PR TITLE
Fix issue 281

### PR DESCRIPTION
**Pull request by aparzi**
Hi @assuncaocharles,
this PR fix issue #281.

### CODE REVIEW

**Comment by aparzi** [comment]
Hi @assuncaocharles,
any update for this PR?

File: projects/ngx-indexed-db/src/lib/ngx-indexed-db.service.ts [hunk 1]
```diff
@@ -305,13 +305,12 @@ export class NgxIndexedDBService {
           const objectStore = transaction.objectStore(storeName);
 
           transaction.oncomplete = () => {
-            const request = objectStore.get(key) as IDBRequest<T>;
-            request.onsuccess = (event: Event) => {
-              obs.next((event.target as IDBRequest<T>).result);
-            };
-            request.onerror = (event: Event) => {
-              obs.error(event);
-            };
```
Comment (assuncaocharles) [inline projects/ngx-indexed-db/src/lib/ngx-indexed-db.service.ts:319]: What about error handling?

Comment (aparzi) [reply projects/ngx-indexed-db/src/lib/ngx-indexed-db.service.ts:319]: Error handling is present in the getByKey method. Other methods also have this management.

Comment (aparzi) [reply projects/ngx-indexed-db/src/lib/ngx-indexed-db.service.ts:319]: Hi @assuncaocharles,
any news?

**Pull request merged**